Makefile

```diff
@@ -281,8 +281,14 @@ deploy-aws-account-operator-credentials: check-aws-credentials
 # Create the aws-account-operator-credentials secret
 	@oc process -p OPERATOR_ACCESS_KEY_ID=${ID} -p OPERATOR_SECRET_ACCESS_KEY=${KEY} -p OPERATOR_NAMESPACE=aws-account-operator -f hack/templates/aws_v1alpha1_aws_account_operator_credentials.tmpl | oc apply -f -
 
+#Deploy the operator configmap
+.PHONY: deploy-operator-configmap
+deploy-operator-configmap:
+# Deploy configmap with stage base and root OUs and account-limit to 1
+	@@oc process -p ROOT=r-0wd6 -p BASE=ou-0wd6-tmsbvahq -p ACCOUNTLIMIT="1" -f hack/templates/aws_v1alpha1_configmap.tmpl | oc apply -f -
```

Comment:
This looks very similar to the existing `create-ou-map` target.

If the idea is to hardcode values for `ROOT` and `BASE`, perhaps this can simply set those values and run that target. However, I kinda suspect that's not the point. So I'll ask another way: is there a reason not to simply reuse that target?